src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetArticlePlaceholderTests.java

```diff
@@ -46,6 +46,7 @@ public void userCanAddVideoFromWikiaUsingPlaceholder() {
 
   @Test(groups = {"VideoArticlePlacehoder_003"})
   @Execute(asUser = User.SUS_STAFF, onWikia = "mobileregressiontesting")
+  @RelatedIssue(issueID = "XW-4584")
```

Comment:
I think this should be fixed on prod already, could you check if test passes and remove this annotation?